portal/patient_self_checkin/step4.php

```diff
@@ -0,0 +1,102 @@
+<?php
```

Comment:
needs a standard license/author/copyright header.(like the one you have in portal/patient_self_checkin/index.php)